Makefile

```diff
@@ -93,15 +95,16 @@ autotest:
 .PHONY: testlong
 testlong:
 	go vet ./...
-	$(GODEP) go test ./...
+	make cover
 	make -C tests test
```

Comment:
The reason I put it in here is that cover needs to run the tests again. At the moment it doesn't matter much as the test suite runs very fast. I see `make test` as the fast option to run the tests and `make testlong` is mainly used on the CI system and to do a final check locally.